backend/ecs_tasks/delete_files/requirements.in

```diff
@@ -1,6 +1,6 @@
 pyarrow==2.0.0
-s3fs==0.4.0
+s3fs==0.5.2
 python-snappy==0.5.4
 pandas==1.1.1
-boto3==1.14.54
+boto3==1.14.44
```

Comment:
downgrading boto3? is this an s3fs compat issue?

Reply:
Note that this is just a draft PR - the downgrade was for s3fs, but a new upgrade made solve.